crates/red_knot_python_semantic/resources/mdtest/type_properties/is_assignable_to.md

```diff
@@ -476,4 +476,22 @@ static_assert(is_assignable_to(CallableTypeOf[keyword_variadic], Callable[..., N
 static_assert(is_assignable_to(CallableTypeOf[mixed], Callable[..., None]))
 ```
 
+### Function types
+
+```py
+from typing import Any, Callable
+
+def f(x: Any) -> str:
+    return ""
+
+def g(x: Any) -> int:
+    return 1
+
+# no error
```

Comment:
I don't think this comments adds any additional context so it might be best to remove it
```suggestion
```